worker/prepare.sh

```diff
@@ -11,12 +11,12 @@ fi
 
 if [[ "$EXTRA_CONDA_PACKAGES" ]]; then
     echo "EXTRA_CONDA_PACKAGES environment variable found.  Installing."
-    /opt/conda/bin/conda install --yes $EXTRA_CONDA_PACKAGES
+    /opt/conda/envs/worker/bin/conda install --yes $EXTRA_CONDA_PACKAGES
```

Comment:
I don't know if there is anything else that being in a conda env does wouldn't another option just be to move the `source activate worker` line to the top of the script? And then just call `conda install` and `pip install`. Not sure if this approach is better?

Reply:
Habit. it has the same effect, and docker couldn't find source so this is how we install all the packages in the Dockerfile. We could use either here.

Reply:
sounds good! In my own troubleshooting I figured out that docker can't find it b/c it uses /bin/sh by default, which doesn't support `source`. If we need it at any point, you can use the command `SHELL ["/bin/bash", "-c"]` prior to calling source. i.e.

```SHELL ["/bin/bash", "-c"]
RUN source activate worker && \
  pip install --src=/ -e git+https://github.com/ClimateImpactLab/clawpack.git#egg=clawpack --no-cache-dir
```